server/database/database.js

```diff
@@ -0,0 +1,78 @@
+const Sequelize = require('sequelize');
+const db = new Sequelize('divvy', 'root', '', {
+  host: 'localhost',
+  dialect: 'mysql',
+
+  pool: {
+    max: 5,
+    min: 0,
+    idle: 10000,
+  },
+}); // name, username, password
+
+db
```

Comment:
I'm not sure this file needs to do the actual authentication. The authentication should probably happen right before the sync inline with starting the server in the `server.js` file would be my gut feeling.

This file would be more of a file to setup the sequelize instance that we would import into files like, `api/user/user.model.js` for example.

Then in the `server.js` we would `authenticate`, `sync`, etc. 

Since we aren't doing that now I think this okay as is and we can refactor when we have a better idea of how to structure and when to initiate a db connection.